libs/ui-watt/src/lib/components/drawer/watt-drawer.component.ts

```diff
@@ -84,25 +87,7 @@ export class WattDrawerComponent {
    * Closes the drawer
    */
   close() {
-    if (this.opened) {
```

Comment:
suggestion: the `if` should be still here because pressing the escape button will still call the method even though the drawer itself is not open because the event is global on the drawer component. That component is still in the DOM.